### PR TITLE
Fix #11: Workbox integration & basic stubs for routing

### LINKED ISSUE #11: Create a ServiceWorker with stubs for all routes in README
We need a ServiceWorker, probably based on https://developers.google.com/web/tools/workbox/, to handle all the routes defined in https://github.com/humphd/next#routes.  For now they can simple stubs that just return static content or data.

### PR DESCRIPTION
**Pull request by Fatehsandhu**
Created basic workbox routing stubs template containing all our ReadMe routing information, will be updated as we progress with the project.

To test this out, pull my branch and do the following:

```bash
npm run build
npm run start
```

### CODE REVIEW

**Comment by Fatehsandhu** [comment]
@dsych Can you please have a look at this and let me know if it will work with your code that you're designing.

**Comment by dsych** [comment]
@Fatehsandhu as far as I can 5rll, your service worker 8s generated by eorkbox-cli, which you later use to inject precache routes from manifest. I don't think I can itegrate changes from #41 because it uses parcel and workbox plugin to bundle my app.
cc @humphd

File: package.json [hunk 1]
```diff
@@ -5,7 +5,9 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" ",
-    "build": "cd website && npm run build"
+    "copy": "copyfiles -u 1 src/**/**/* src/**/* src/* website",
+    "build": "cd website&& npm run copy && workbox injectManifest workbox-config.js && npm run build",
```
Comment (dsych) [inline package.json]: Why are you running build script inside build script? Wouldn't it create a circular dependency?

File: package.json [hunk 2]
```diff
@@ -21,6 +23,12 @@
     "docusaurus": "^1.0.15",
     "husky": "^0.14.3",
     "precise-commits": "^1.0.2",
-    "prettier": "^1.12.1"
+    "prettier": "^1.12.1",
+    "copyfiles": "1.2.0",
+    "workbox-cli": "^3.0.1"      
+  },
+  "dependencies": {
+    "express": "4.15.4",
```
Comment (dsych) [inline package.json]: Why are we running a server? Isn't the whole point of the project to go serverless?

Comment (Fatehsandhu) [reply package.json]: I removed it

Comment (dsych) [reply package.json]: As far as I can tell, it is still there.

Comment (Fatehsandhu) [reply package.json]: Which line are you referring to?

Comment (dsych) [reply package.json]: express dependency

File: package.json [hunk 3]
```diff
@@ -21,6 +22,11 @@
     "docusaurus": "^1.0.15",
     "husky": "^0.14.3",
     "precise-commits": "^1.0.2",
-    "prettier": "^1.12.1"
+    "prettier": "^1.12.1",
+    "copyfiles": "1.2.0",
+    "workbox-cli": "^3.0.1"      
+  },
+  "dependencies": {
+    "npm": "^6.0.1"
```
Comment (dsych) [inline package.json]: Npm is not really a dependency unless you are interacting with it's code.

**Comment by Fatehsandhu** [comment]
Merged with the PR# 41

**Pull request closed without merging**